src/mapping/mappers/SyntheaToV09.js

```diff
@@ -1,31 +1,6 @@
 const { AggregateMapper } = require('../mapper');
 const { applyProfile, applyProfileFunction, hasProfileFromList, mcodeUtils09 } = require('../../utils');
 
-const defaultProfile = (resourceType) => {
```

Comment:
Do you need to use `mcodeUtils09.defaultProfile` in this mapper now that the defaultProfile was moved out?

Reply:
yiiiikes dang great catch. Resolved